### ISSUE TITLE
REST Query URI when using .basePath()

### ISSUE THREAD

**Issue by pnsantos** [serenity-bdd/serenity-core#300]
Hi,

I'm currently using Serenity's rest assured to make API calls, but it seems that the html report doesn't include the contents of basePath in the URI part of the request. For example:
```
rest().given()
        .filter(someFilter)
        .baseUri("http://some.host/")
        .basePath("path/to/resource")
    .when()
        .get()
    .then()
        ...
```
The report for this request will have the URI part empty in the report. This is because no path was passed to `get()`. Wouldn't it make for sense to show the base path + whatever is passed in the `get()` as the requested URI path?)

Now the reason I'm doing it like this is to be able to access the resource path in the filter (not sure if I can access what is passed in the `get()` I don't think it's possible but if there is a way to do it I'd appreciate if somebody could tell me how).

**Comment by YamStranger** [comment]
Hello @pnsantos Thanks for this issue!!! 

I am not sure about filters and working them with "basePath". 
But I see your issue about using basePath and it should be fixed, thanks!!

**Comment by pnsantos** [comment]
Awesome, thanks!

**Comment by YamStranger** [comment]
depends on #355

**Comment by YamStranger** [comment]
Hello @pnsantos

You issue is fixed and can be checked in serenity-core 1.1.31. 

Now you can make configuration of request using sample request specification and as you described in this issue using baseURI and basePath. 

To check it I have created method, where called configuration methods in same way as you: 

```
    public void whenIAddThePetToTheStore() {
        for (Pet pet : pets) {
            int id = Math.abs(new Random().nextInt());
            Map<String, Object> jsonAsMap = new HashMap<>();
            jsonAsMap.put("id", id);
            jsonAsMap.put("name", pet.getName());
            jsonAsMap.put("status", pet.getStatus());
            jsonAsMap.put("photoUrls", new ArrayList<>(Arrays.asList()));

            given().contentType("application/json")
                    .content(jsonAsMap).log().body()
                    .baseUri("http://petstore.swagger.io")
                    .basePath("v2/pet")
            .when().post();

            pet.setId(id);
        }
    }
```

Report now include full path as expected, 
![before3](https://cloud.githubusercontent.com/assets/1667699/14665571/c23fa1be-06dc-11e6-84b3-6e36979ee594.png)

**Comment by pnsantos** [comment]
Thanks @YamStranger this is great
